src/interfaces/filing-history-list-state-interface.ts

```diff
@@ -1,4 +1,5 @@
 import { ApiFilingIF } from '@/interfaces'
+import { FilingRegistraionIF } from './filing-registration-interface'
```

Comment:
Typo.

And please remove commas after properties in the interfaces you touch. Thx.